### ISSUE TITLE
You accepted my changes to make a PowerPlatform version of the workflow tools, but there is no release for the solution

### ISSUE THREAD

**Issue by james-tekstack** [demianrasko/Dynamics-365-Workflow-Tools#263]
Hi Demian,

You recently approved and completed my pull request that added a new Visual Studio solution without any dependencies on D365. The results is a generic, Power Platform version of your awesome Workflow Tools. However, your recent releases did not include solution files for the Power Platform-only version. 

Going forward, can you build and release a Power Platform Workflow Tools solution in addition to the solution for D365? The Visual Studio solution file is already part of your repo. You just need to use it to create a Power Platform version of the solution.

Thanks for an awesome product. Keep up the good work.

**Comment by RicLund** [comment]
Yes agreed @james-tekstack - we need these awesome tools when not using D365. @demianrasko is this possible?

**Comment by mathiaslaegdsmand** [comment]
Agree - would be great to have the CDS or Power Platform version. @demianrasko can you create this for the latest release so it is available for download also as a managed package?
